packages/@expo/config-plugins/src/utils/generateCode.ts

```diff
@@ -10,8 +10,8 @@ function getGeneratedSectionIndexes(
   tag: string
 ): { contents: string[]; start: number; end: number } {
   const contents = src.split('\n');
-  const start = contents.findIndex((line) => line.includes(`@generated begin ${tag}`));
-  const end = contents.findIndex((line) => line.includes(`@generated end ${tag}`));
+  const start = contents.findIndex((line) => RegExp(`@generated begin ${tag} -`).test(line));
```

Comment:
This one I didn't need to replace `line.includes`, I just wanted to be consistent with the next line.